### PR TITLE
Feature/cvsb-6944

### PR DESCRIPTION
**Pull request by oalexandru**

### CODE REVIEW

File: tests/unit/reportGen.unitTest.ts [hunk 1]
```diff
@@ -186,15 +189,108 @@ describe("report-gen", () => {
...
+                                });
+                        });
+                });
+
+            });
+
+
+            context("the report contains prohibitionIssued false on testType level and false on defects level", () => {
+                it("should contain on the corresponding testType line, on the failureAdvisoryItemsQAICommentsTestValue column, the info that the prohibition WAS" +
+                    " issued on defects level and that the prohibition WAS also issued on the Additional test type notes level", () => {
```
Comment (bsiddappa) [inline tests/unit/reportGen.unitTest.ts]: Guess the descriptions are not in sync. Will need to be changed to Not issued and none for additional notes.

File: tests/unit/reportGen.unitTest.ts [hunk 2]
```diff
@@ -186,15 +189,108 @@ describe("report-gen", () => {
...
+                                    expect (failureAdvisoryItemsQAICommentsTestValue).to.contain("Additional test type notes: none;");
+                                });
+                        });
+                });
+
+            });
+
+            context("the report contains prohibitionIssued true on testType level and false on defects level", () => {
+                it("should contain on the corresponding testType line, on the failureAdvisoryItemsQAICommentsTestValue column, the info that the prohibition WAS" +
+                    " issued on defects level and that the prohibition WAS also issued on the Additional test type notes level", () => {
```
Comment (bsiddappa) [inline tests/unit/reportGen.unitTest.ts]: shouldnt it be not issued?

File: tests/unit/reportGen.unitTest.ts (hunk 1 above)
Comment (oalexandru) [reply tests/unit/reportGen.unitTest.ts]: fixed

File: tests/unit/reportGen.unitTest.ts (hunk 2 above)
Comment (oalexandru) [reply tests/unit/reportGen.unitTest.ts]: fixed

**Pull request merged**